src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/change_password.html.twig

```diff
@@ -0,0 +1,100 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% set passwordHelpMessage = 'Password should be at least %num% characters long.'|trans({ '%num%': 8 }, 'Admin.Advparameters.Help') %}
```

Comment:
```suggestion

{% set passwordMinLength = constant('PrestaShop\\PrestaShop\\Core\\Domain\\Employee\\ValueObject\\Password::MIN_LENGTH') %}
{% set passwordHelpMessage = 'Password should be at least %num% characters long.'|trans({ '%num%': passwordMinLength }, 'Admin.Advparameters.Help') %}
```

Reply:
This is nice, didn't know you could add constant like this. Thanks!

Reply:
I just copied what has been done here https://github.com/PrestaShop/PrestaShop/pull/22045/files#diff-dbc3e4500f8b7833c72212c5fa8c5e8c40e1edf2a5a168bdfb722474d3514e81R71 😄